largest_prime_factor.py

```diff
@@ -0,0 +1,27 @@
+"""
+The prime factors of 13195 are 5, 7, 13 and 29.
+
+What is the largest prime factor of the number 600851475143? 
+"""
+def is_prime(n):
+	if n <= 2:
+		return False
+	for x in range(2, num1):
```

Comment:
did you run this, num1 looks like it's undefined here

Reply:
this part did not run but I did fix that part